pkg/cluster/sync.go

```diff
@@ -34,6 +35,16 @@ func (c *Cluster) Sync(newSpec *acidv1.Postgresql) error {
 		}
 	}()
 
+	if err = c.validateResources(&c.Spec); err != nil {
```

Comment:
I assume the min resource requirement affects only `postgres` pods and not Scalyr and other sidecars ?

Reply:
yes. Is it not clear from the code?